app/views/hyrax/file_sets/_permission.html.erb

```diff
@@ -1,5 +1,5 @@
   <div id="permissions_display" class="tab-pane">
-    <%= simple_form_for [main_app, file_set],
+    <%= simple_form_for [main_app, @form],
```

Comment:
Making this simple substitution “works” in the sense that it gets the page to render, but it doesn’t actually “work” in the sense that if you click save (in Koppie) it throws an error :P . I’m not sure if just passing `@form` here is a naive approach, or if it is correct and something else in Koppie is broken.